HOLD FOR RELEASE: Adds adoption rate docs

paigecalvert opened the pull request:
New topic about adoption rate: https://deploy-preview-1052--replicated-docs.netlify.app/vendor/customer-adoption

Links to adoption rate topic from the About Customer topic (since adoption rate shows up on the customers page): https://deploy-preview-1052--replicated-docs.netlify.app/vendor/licenses-about

Made edits to this topic to have it serve as an "overview" (a home for Limitations, a description of the instance check-in events, and details about how the vendor portal creates events): https://deploy-preview-1052--replicated-docs.netlify.app/vendor/instance-insights-event-data

Other small changes to reorganize or other update related content in other topics.

paigecalvert commented on docs/vendor/instance-insights-details.md:
```diff
@@ -3,37 +3,6 @@ import Checkins from "../partials/instance-insights/_appCheckins.mdx"
 
 This topic describes using the Replicated vendor portal to quickly understand the recent events and performance of application instances installed in your customers' environments.
-
-## About Instance Reporting {#about-reporting}
```
Moved this information to https://deploy-preview-1052--replicated-docs.netlify.app/vendor/instance-insights-event-data, which now serves as a top level "overview" topic

paigecalvert commented on docs/vendor/instance-insights-details.md:
```diff
@@ -3,37 +3,6 @@ import Checkins from "../partials/instance-insights/_appCheckins.mdx"
-
-The vendor portal also uses this data to display insights about the active instances of your application. For more information about the instance data insights displayed in the vendor portal, see [About the Instance Details Page](#about-page) below.
-
-## Requirements and Limitations
```
Also moved Limitations over to https://deploy-preview-1052--replicated-docs.netlify.app/vendor/instance-insights-event-data

paigecalvert commented on docs/vendor/instance-insights-event-data.md:
```diff
@@ -1,8 +1,21 @@
 This topic describes the application instance data fields that the Replicated vendor portal uses to generate events for the instance.
 
-## About Events
+## How the Vendor Portal Collects Instance Data {#about-reporting}
```
Update headings to How the Vendor Portal Collects Instance Data and How the Vendor Portal Generates Events to be more descriptive

jonquil2002 replied:
Love this.

paigecalvert commented on sidebars.js:
```diff
@@ -235,11 +235,12 @@ const sidebars = {
     {
       type: 'category',
-      label: 'Reporting',
+      label: 'Insights and Reporting',
```
Insights and Reporting felt more descriptive than just Reporting

paigecalvert commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,91 @@
+# Adoption Rate
```
All new content to describe adoption rate data in its own topic.

paigecalvert commented on docs/vendor/licenses-about.md:
```diff
@@ -76,11 +76,14 @@ The Replicated vendor portal **Customers** page displays details about each cust
 [View a larger version of this image](/images/customer-page.png)
 
 From the **Customers** page, you can do the following:
+* View insights about the adoption rate of each version of your application across your customers. For more information, see [Adoption Rate](customer-adoption).
```
Updates the [About Customers](https://deploy-preview-1052--replicated-docs.netlify.app/vendor/licenses-about) topic to include reference to the new Adoption rate section on the Customers page

paigecalvert commented on docs/vendor/instance-insights-event-data.md:
```diff
@@ -17,7 +30,20 @@ Each event that the vendor portal generates for application instances has the fo
+* **Air gap not supported**: Instance data is available only for application instances installed in online environments. Data for instances installed in air gapped environments is not available.
+* **Active instances only**: Instance data is available only for active application instances. An instance is considered inactive when its most recent check-in was more than two weeks ago. An instance can become inactive if it is decommissioned, stops checking for updates, or otherwise stops reporting.
+
+   The vendor portal continues to display data for an inactive instance from its most-recently seen state. This means that data for an inactive instance might continue to show a Ready status after the instance becomes inactive. Replicated recommends that you use the timestamp in the **Last Check-in** field to understand if an instance might have become inactive, causing its data to be out-of-date ...
```
I updated the Active Instances Only limitation to clarify that active = check in within the last two weeks. This is a figure that I saw in the working backwards doc for customer rollups https://docs.google.com/document/d/1XxJURXxBCjTgScgvHmFuC7qdUvk6nMrdWRLeA57jZ38/edit#

dexhorthy replied:
@paigecalvert I think this is fixed and @divolgin can confirm

paigecalvert commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,91 @@
+    <tr>
+      <th>Metric</th>
+      <th>Description</th>
+      <th>Recommended Target Value</th>
```
Testing out a column here where we describe the recommended target value that they should be shooting for. I thought this might be useful info, and it would also then imply which arrow direction is "good/bad".

Just wasn't sure if "target value" is something we're prepared to offer

paigecalvert replied:
Perhaps we could also add more general suggestions, like the lower/higher the better

paigecalvert commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,105 @@
+
+Based on the time period selected, each metric includes an arrow that shows the change in value compared to the previous period. For example, if the median relative age today is 68 days, the selected time period is three months, and three months ago the median relative age was 55 days, then the metric would show an upward-facing arrow with an increase of 13 days. 
+
+The following table describes each metric in the **Adoption rate** section, including the formula used to calculate its value and the recommended trend for the metric over time: 
```
I added a column in this table for "Target Trend" to explain the target change in time for each metric.

paigecalvert commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,105 @@
+
+The following shows an example of an adoption rate graph with three months of data:
+
+![Adoption rate graph showing three months of data](/images/adoption_rate_graph.png)
```
@dexhorthy I was thinking it could be useful to show the popup that appears when you hover over the graph as part of this screenshot, but for whatever reason I wasn't able to access the wordpress app that you shared with me. (I don't think this is a blocker for release or anything)

dexhorthy replied:
I just reinvited `paige+wpe@replicated.com` to a demo account

paigecalvert commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,109 @@
+
+:::note
+The **Adoption rate** section is Beta. To enable this feature on your account, log in to your vendor portal account, select **Support > Request a feature**, and submit a feature request for "Adoption rate".
+:::
```
@dexhorthy to correct this notice with the actual name of the feature flag that they should request

paigecalvert commented on docs/vendor/licenses-about.md:
```diff
@@ -81,6 +81,7 @@ The following shows an example of the **Customers** page:
 [View a larger version of this image](/images/customer-page.png)
 
 From the **Customers** page, you can do the following:
+<!-- * View insights about the adoption rate of each version of your application across your customers. For more information, see [Adoption Rate (Beta)](customer-adoption). -->
```
@dexhorthy FYI commented out this link from the main About Customers topic to the new adoption rate topic for now since the feature is Beta.

Would also need an updated screenshot of the full customers page showing the adoption rate section in this topic as well

dexhorthy commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,109 @@
+# Adoption Rate (Beta)
```
```suggestion
# Adoption Report (Beta)
```

dexhorthy commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,109 @@
+# Adoption Rate (Beta)
+
+This topic describes the insights in the **Adoption rate** section on the Replicated vendor portal **Customers** page.
```
```suggestion
This topic describes the insights in the **Adoption** section on the Replicated vendor portal **Customers** page.
```

dexhorthy commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,109 @@
+
+## About Adoption Rate
+
+The **Adoption rate** section on the **Customers** page provides insights about the rate at which your customers upgrade their instances and adopt the latest versions of your application. As an application vendor, you can use these adoption rate metrics to learn if your customers are completing upgrades regularly, which is a key indicator of the discoverability and ease of application upgrades.
```
```suggestion
The **Adoption** section on the **Customers** page provides insights about the rate at which your customers upgrade their instances and adopt the latest versions of your application. As an application vendor, you can use these adoption rate metrics to learn if your customers are completing upgrades regularly, which is a key indicator of the discoverability and ease of application upgrades.
```

dexhorthy commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,109 @@
+
+The **Adoption rate** section on the **Customers** page provides insights about the rate at which your customers upgrade their instances and adopt the latest versions of your application. As an application vendor, you can use these adoption rate metrics to learn if your customers are completing upgrades regularly, which is a key indicator of the discoverability and ease of application upgrades.
+
+The vendor portal generates adoption rate data from all of your customer's active application instances. An instance is considered _active_ if no more than two weeks have passed since its last check-in event. For more information about instance check-ins, see [How the Vendor Portal Collects Instance Data](instance-insights-event-data#about-reporting) in _About Instance and Event Data_.
```
@divolgin is this note about "active" instances true? Or do we only collect data from instances that checked in within the period? My guess is that the language here can be simplified a bit.

divolgin replied:
We show data for instances that checked in the given time period.

The adoption rates on the channel cards themselves only show instances active within the last 24 hours.

paigecalvert replied:
Gotcha, so as long as they had a check-in one time in the given time period. Agreed Dex I can simplify this (not worth defining active here).

dexhorthy commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,109 @@
+
+The vendor portal generates adoption rate data from all of your customer's active application instances. An instance is considered _active_ if no more than two weeks have passed since its last check-in event. For more information about instance check-ins, see [How the Vendor Portal Collects Instance Data](instance-insights-event-data#about-reporting) in _About Instance and Event Data_.
+
+The following screenshot shows an example of the **Adoption rate** section:
```
```suggestion
The following screenshot shows an example of the **Adoption** report:
```

dexhorthy commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,109 @@
+
+[View a larger version of this image](/images/customer_adoption_rates.png)
+
+As shown in the screenshot above, the **Adoption rate** section includes a graph and key adoption rate metrics. For more information about how to interpret this data, see [Adoption Rate Graph](#graph) and [Adoption Rate Metrics](#metrics) below.
```
```suggestion
As shown in the screenshot above, the **Adoption** report includes a graph and key adoption rate metrics. For more information about how to interpret this data, see [Adoption Rate Graph](#graph) and [Adoption Rate Metrics](#metrics) below.
```

dexhorthy commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,109 @@
+
+As shown in the screenshot above, the **Adoption rate** section includes a graph and key adoption rate metrics. For more information about how to interpret this data, see [Adoption Rate Graph](#graph) and [Adoption Rate Metrics](#metrics) below.
+
+The **Adoption rate** section also displays the number of customers assigned to the selected channel and a link to the report that you can share with other members of your team.
```
```suggestion
The **Adoption** report also displays the number of customers assigned to the selected channel and a link to the report that you can share with other members of your team.
```

dexhorthy commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,109 @@
+
+The **Adoption rate** section also displays the number of customers assigned to the selected channel and a link to the report that you can share with other members of your team.
+
+You can filter the graph and metrics in the **Adoption rate** section by:
```
```suggestion
You can filter the graph and metrics in the **Adoption** report by:
```

dexhorthy commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,109 @@
+
+## Adoption Rate Graph {#graph}
+
+The **Adoption rate** section includes a graph that shows the percent of active instances that are running different versions of your application on each day within the selected time period.
```
```suggestion
The **Adoption** report includes a graph that shows the percent of active instances that are running different versions of your application on each day within the selected time period.
```

dexhorthy commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,109 @@
+
+[View a larger version of this image](/images/adoption_rate_graph.png)
+
+As shown in the image above, the graph plots the number of active instances on each day in the selected time period. Additionally, that graph uses colors to represent the application versions that were active on each day. The key to the left of the graph shows the unique color that is assigned to each application version. You can use this color-coding to see at a glance the percentage of active i ...
```
```suggestion
As shown in the image above, the graph plots the number of active instances in each week in the selected time period. Additionally, the graph uses colors to represent the application versions that were active in each week. The key to the left of the graph shows the unique color that is assigned to each application version. You can use this color-coding to see at a glance the percentage of active instances that were running different versions of your application across the selected time period.
```

dexhorthy commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,109 @@
+
+As shown in the image above, the graph plots the number of active instances on each day in the selected time period. Additionally, that graph uses colors to represent the application versions that were active on each day. The key to the left of the graph shows the unique color that is assigned to each application version. You can use this color-coding to see at a glance the percentage of active i ...
+
+You can also hover over a color-coded section in the graph to view the number and percentage of active instances that were running the version on the given day.
```
```suggestion
You can also hover over a color-coded section in the graph to view the number and percentage of active instances that were running the version in a given period.
```

jonquil2002 commented on docs/vendor/instance-insights-event-data.md:
```diff
@@ -17,7 +30,20 @@ Each event that the vendor portal generates for application instances has the fo
+* **Instance data freshness**: The rate at which data is updated in the vendor portal varies depends on how often the vendor portal receives instance data from the app manager. The vendor portal receives instance data when any of the following occur:
+  <Checkins/>
+* **Event timestamps**: The timestamp of events displayed on the **Instances details** page is the timestamp when the Replicated Vendor API received the instance data from the app manager. The timestamp of events does not necessarily reflect the timestamp of when the event occurred.
+* **Caching for Kubernetes Installer Cluster Data**: For clusters created with the Replicated Kubernetes Installer, the app manager stores the counts of total nodes and ready nodes in a cache for five minutes. If the app manager sends instance data to the vendor portal within the five minute window, then the reported data for total nodes and ready nodes reflects the data in the cache. This means  ...
```
This bold content is using title case but the other items in the list are using sentence case.

jonquil2002 replied:
Also, in the text portion of in, I think `Kubernetes Installer` should be `Kubernetes installer` (lower case) because it's just a component/feature.

dexhorthy commented on docs/vendor/customer-adoption.md:
```diff
@@ -0,0 +1,109 @@
+This topic describes the insights in the **Adoption** section on the Replicated vendor portal **Customers** page.
+
+:::note
+The **Adoption Report** section is currently in Beta -- you will see it labeled as such in the Vendor Portal. the character of this data and how it is computed and displayed is subject to change.
```
```suggestion
The **Adoption Report** section is currently in Beta -- you will see it labeled as such in the Vendor Portal. The character of this data, how it is computed, and how it is displayed is subject to change.
```

paigecalvert commented:
@dexhorthy just an FYI that I fixed the merge conflict and also updated the screenshot on the customers page to include the adoption rate/instance details update per our convo yesterday. this is ready whenever!
